lib/vrl/stdlib/src/parse_datadog_grok.rs

```diff
@@ -0,0 +1,326 @@
+use datadog_grok::{
```

Comment:
From what I can see, this file isn't being used? Can we remove it?

Reply:
Oh, right, indeed, thanks 👍